### DISCUSSION TITLE
File changes have no effect

### DISCUSSION THREAD

**Discussion by tagdo** [ddev/ddev#5522] [Q&A]
Hi, 

I have a problem with my ddev and TYPO3 version 11 installation. I clear all TYPO3 caches after file changes that have no effect, until ddev composer dumpautoload -a and ddev composer clearcache. But this seems that something doesn't work right, because it takes a long time and it is not usual way. Can anybody help me in that case?

Thanks in advance.

Ayhan

**Comment by rfay** [comment]
Hi @tagdo - Can you please give us more information by doing `ddev debug test` and providing the output of that as a gist (gist.github.com) or as a file attached here? It may be better to ask about this in TYPO3 slack, I don't think others are having trouble, it sounds like probably a TYPO3 behavior?
